Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM centos:6
+
+RUN yum update -y && yum install -y wget
+RUN wget http://ftp.rediris.es/mirror/fedora-epel/6/i386/epel-release-6-8.noarch.rpm && yum localinstall -y --nogpgcheck epel-release-6-8.noarch.rpm
+RUN yum install -y npm git
```

Comment:
You may want to do only one RUN command in this file. This saves up image bytes big time.